test.sh

```diff
@@ -74,6 +74,26 @@ fi
 DATABASE_USER="postgres"
 DATABASE_PASSWORD="postgres-password"
 
+function dump_debug_info() {
+  announce "Showing summary of Kubernetes objects in $CONJUR_NAMESPACE namespace"
+  kubectl get all -n "$CONJUR_NAMESPACE"
+
+  announce "Showing Conjur OSS Server logs"
+  kubectl logs -l "app=conjur-oss" -c conjur-oss
```

Comment:
should this be -c $RELEASE_NAME as below?

Reply:
Good question... but we should be okay. The container name for the Conjur server container will be "conjur-oss" (it's actually set to `.Chart.Name` by Helm, but we never change that):
https://github.com/cyberark/conjur-oss-helm-chart/blob/main/conjur-oss/templates/deployment.yaml#L100

Whereas the NGINX container uses `{{ .Release.Name }}-nginx`:
https://github.com/cyberark/conjur-oss-helm-chart/blob/main/conjur-oss/templates/deployment.yaml#L59

We definitely should change the Conjur OSS Helm charts to use FIXED container names. The container names are only scoped to the Pod, so there's no need to add Release names or Chart names to make these unique... i.e. they're guaranteed to be unique within the realm of the Pod.